doc/README.md

```diff
@@ -191,33 +194,34 @@ paths with the highest cum weight.
   browser to view it.
 * **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
 
-### Interpreting the Callgraph
+### Interpreting the Nodes
+
+![callgraph1](images/callgraph_nodes.svg)
 
-*   **Total Value**: amount of resource used by location and all location it
-    calls.
+*  _cum_ value is indiated by the nodes' color.
 
     *   Redder nodes have larger total values.
     *   Greyer nodes have smaller total values.
 
-*   **Self Value**: amount of resource used by lines of code within the
-    location.
+* _flat_ value is indicated by the font size.
 
     *   Nodes with larger text have larger self values.
     *   Nodes with smaller text have smaller self values.
 
-![callgraph1](images/callgraph1.svg)
+
+### Interpreting the Edges 
+
+![callgraph2](images/callgraph_edges.svg)
 
 *   **Dashed Edges**: some locations between two connected locations were
-    pruned.
+    removed.
 
 *   **Solid Edges**: one location directly calls the other.
 
 *   **Thicker & Redder Edges**: more resources were along that path.
```

Comment:
Nit (here and next line): add "used" (more resources were used along that path).

Rephrasing here might be better. `pprof ` can be used to view any profile.proto, so defining things in terms of resource usage may not be ideal. If this can be rephrased to use "value" instead, that might be better.

Reply:
Done.